pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>su.litvak.chromecast</groupId>
     <artifactId>api-v2</artifactId>
     <version>0.9.2-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```

Comment:
Do you know that the artifact is still the "same" of type jar but with a OSGi compatible manifest?
We do not change the artifact type here, only grant that the maven-bundle-plugin of Apache Felix is used for packaging (and it could add a manifest by inspection of the classes).

We could also leave the packaging type as it is.
See "Adding OSGi metadata to existing projects without changing the packaging type" here "http://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html"

But could you (please) explain me your concerns as long as the resulting artifact type is not changed?